src/FSharp.AWS.DynamoDB/Expression/ExprCommon.fs

```diff
@@ -48,19 +50,19 @@ type AttributeId =
             append id.RootId
 
             for nf in id.NestedAttributes do
-                append <| nf.Print())
+                append <| nf.Print ())
 
     member id.Name =
         mkString (fun append ->
             append id.RootId
 
             for nf in id.NestedAttributes do
-                append <| nf.Print())
+                append <| nf.Print ())
```

Comment:
this is Wrong - never a space in a Method call - some cases don't compile correctly either if you do that
(always before a function arg)

Reply:
Will turn back off, was trying to fix the DU scenario